packages/react-composites/src/composites/CallComposite/MockCallAdapter.ts

```diff
@@ -98,7 +98,7 @@ export class MockCallAdapter implements CallAdapter {
     throw Error('createStreamView not implemented');
   }
   disposeStreamView(): Promise<void> {
-    throw Error('disposeStreamView not implemented');
+    return Promise.resolve();
```

Comment:
why do we need an implementation here now? is there one needed for createView as well?

Reply:
One of the tests called it as the tile logic now completes its unmount before the test finishes

Reply:
ahhh okay that makes sense